lib/storage_tables/attachable/model.rb

```diff
@@ -43,6 +43,42 @@ def stored_one_attachment(name, class_name:)
           )
           ActiveRecord::Reflection.add_attachment_reflection(self, name, reflection)
         end
+
+        def stored_many_attachments(name, class_name:)
+          define_method(name) do
+            @storage_tables_attached ||= {}
+            @storage_tables_attached[name.to_sym] ||= StorageTables::Attachable::Many.new(name.to_s, self)
+          end
+
+          define_method(:"#{name}=") do |attachables|
+            attachables = Array(attachables).compact_blank
+            pending_uploads = attachment_changes[name.to_s].try(:pending_uploads)
+
+            attachment_changes[name.to_s] = if attachables.none?
+              StorageTables::Attachable::Changes::DeleteMany.new(name.to_s, self)
+            else
+              StorageTables::Attachable::Changes::CreateMany.new(name.to_s, self, attachables, pending_uploads:)
+            end
+          end
+
+          has_many(:"#{name}_storage_attachments", class_name: class_name.to_s, inverse_of: :record,
+                                                   foreign_key: :record_id)
+          has_many(:"#{name}_storage_blobs", through: :"#{name}_storage_attachments",
+                                             class_name: "StorageTables::Blob", source: :blob)
+
+          scope :"with_attached_#{name}", -> { includes("#{name}_attachments": :blob) }
```

Comment:
Weet niet waar deze allemaal voor gebruikt wordt, maar ik denk dat we met onze opzet eigenlijk altijd zonder de blob kunnen.

```suggestion
          scope :"with_attached_#{name}", -> { includes(:"#{name}_attachments") }
```